Build/start.sh

```diff
@@ -52,6 +52,12 @@ MACHINE_FEATURES_append = " docker"
 DISTRO_FEATURES_append = " virtualization"
 EOT
 
+# Set default root password
+#cat <<EOT >> ${LOCALCONF}
+echo "inherit extrausers" >> ${LOCALCONF}
```

Comment:
It needs to be `echo "INHERIT += \"extrausers\" >> ${LOCALCONF}`

Reply:
Modified it to:
```
INHERIT += "extrausers"
EXTRA_USERS_PARAMS = "usermod -P root root"
```
and it built fine on dc02 for my user (`reggler`)